From: Michael Froman <mjfroman@mac.com>
Date: Wed, 3 Jan 2024 12:23:39 -0600
Subject: (tmp-cherry-pick) Revert "Roll chromium_revision
 e9af340c3f..84734fb4aa (1215440:1215550)" (3f64824b73)

This reverts commit 0e00acec9f962e70e7255e55f3514905dbc4c8eb.

Reason for revert: See https://webrtc-review.googlesource.com/c/src/+/325241.

Original change's description:
> Roll chromium_revision e9af340c3f..84734fb4aa (1215440:1215550)
>
> Change log: https://chromium.googlesource.com/chromium/src/+log/e9af340c3f..84734fb4aa
> Full diff: https://chromium.googlesource.com/chromium/src/+/e9af340c3f..84734fb4aa
>
> Changed dependencies
> * src/ios: https://chromium.googlesource.com/chromium/src/ios/+log/2b80524b98..dd37c54367
> * src/testing: https://chromium.googlesource.com/chromium/src/testing/+log/39e066e15c..cd988cac45
> * src/third_party: https://chromium.googlesource.com/chromium/src/third_party/+log/317f30cbb5..a2aecd446f
> * src/third_party/androidx: TNNVRr7zAcn3PkRswu2uYXsb50DRWyDPtvsbYbVBQ5oC..S2mTZLxkPp9yV9lixw-NGMad2Qv7hpI5zjIBJuEBGl8C
> * src/third_party/perfetto: https://android.googlesource.com/platform/external/perfetto.git/+log/13ce0c9e13..4a8e4a6256
> * src/third_party/r8: hCR0xJbBeRfCUH-G2O_dMQ2C7wY-BhWHhAdXP_yuG3MC..Jn6jDwY2CaSHjf9fzclZsEGDIaIudbGyiQAiqu6fjnMC
> * src/tools: https://chromium.googlesource.com/chromium/src/tools/+log/c5b94a6e79..c06198bfac
> DEPS diff: https://chromium.googlesource.com/chromium/src/+/e9af340c3f..84734fb4aa/DEPS
>
> No update to Clang.
>
> BUG=None
>
> Change-Id: Idde409044c84ce8381cf25e5588a8218ee5538a1
> Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/325200
> Bot-Commit: Autoroller <chromium-webrtc-autoroll@webrtc-ci.iam.gserviceaccount.com>
> Commit-Queue: Autoroller <chromium-webrtc-autoroll@webrtc-ci.iam.gserviceaccount.com>
> Cr-Commit-Position: refs/heads/main@{#41017}

BUG=None

No-Presubmit: true
No-Tree-Checks: true
No-Try: True
Change-Id: Ia82be113c1bc34daba70d9a825393b11b9cd3f2e
Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/325242
Commit-Queue: Mirko Bonadei <mbonadei@webrtc.org>
Bot-Commit: rubber-stamper@appspot.gserviceaccount.com <rubber-stamper@appspot.gserviceaccount.com>
Cr-Commit-Position: refs/heads/main@{#41022}
---
 DEPS | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/DEPS b/DEPS
index 78e5cb0704..2d0f371c47 100644
--- a/DEPS
+++ b/DEPS
@@ -10,7 +10,7 @@ vars = {
   # chromium waterfalls. More info at: crbug.com/570091.
   'checkout_configuration': 'default',
   'checkout_instrumented_libraries': 'checkout_linux and checkout_configuration == "default"',
-  'chromium_revision': '84734fb4aa429ddc3f0ca310f62a4b43919dca3c',
+  'chromium_revision': 'e9af340c3f9945de4039a24d546d0f46f004302b',
 
   # Fetch the prebuilt binaries for llvm-cov and llvm-profdata. Needed to
   # process the raw profiles produced by instrumented targets (built with
@@ -61,13 +61,13 @@ deps = {
     'condition': 'checkout_android',
   },
   'src/ios': {
-    'url': 'https://chromium.googlesource.com/chromium/src/ios@dd37c5436744a53ae039f3883593ca7809496d36',
+    'url': 'https://chromium.googlesource.com/chromium/src/ios@2b80524b9852acd18bff8c8d5d5bb099b4e0bb27',
     'condition': 'checkout_ios',
   },
   'src/testing':
-    'https://chromium.googlesource.com/chromium/src/testing@cd988cac45aa2048b95809e8a5fd7f2b27f8a168',
+    'https://chromium.googlesource.com/chromium/src/testing@39e066e15cfae135d6fbdb5db63ffaa85e69d6e2',
   'src/third_party':
-    'https://chromium.googlesource.com/chromium/src/third_party@a2aecd446fb7ed41d867bd344b9ce6a83d4d34d2',
+    'https://chromium.googlesource.com/chromium/src/third_party@317f30cbb505acaef85f6310c6e18ffb9293187a',
 
   'src/buildtools/linux64': {
     'packages': [
@@ -289,7 +289,7 @@ deps = {
       'condition': 'checkout_android',
   },
   'src/third_party/perfetto':
-    'https://android.googlesource.com/platform/external/perfetto.git@4a8e4a6256e16d82f36baad6fd9e0075ce6d0904',
+    'https://android.googlesource.com/platform/external/perfetto.git@13ce0c9e13b0940d2476cd0cff2301708a9a2e2b',
   'src/third_party/libvpx/source/libvpx':
     'https://chromium.googlesource.com/webm/libvpx.git@424723dc025ce451dab9568239a46b18d0919b4d',
   'src/third_party/libyuv':
@@ -314,7 +314,7 @@ deps = {
       'packages': [
           {
               'package': 'chromium/third_party/r8',
-              'version': 'Jn6jDwY2CaSHjf9fzclZsEGDIaIudbGyiQAiqu6fjnMC',
+              'version': 'hCR0xJbBeRfCUH-G2O_dMQ2C7wY-BhWHhAdXP_yuG3MC',
           },
       ],
       'condition': 'checkout_android',
@@ -338,7 +338,7 @@ deps = {
     'condition': 'checkout_android',
   },
   'src/tools':
-    'https://chromium.googlesource.com/chromium/src/tools@c06198bfac5b514743a699079b557b4fb5e1a64e',
+    'https://chromium.googlesource.com/chromium/src/tools@c5b94a6e79108bd3811b1075dffcf306c18aa13e',
 
   'src/third_party/accessibility_test_framework': {
       'packages': [
@@ -410,7 +410,7 @@ deps = {
     'packages': [
       {
           'package': 'chromium/third_party/androidx',
-          'version': 'S2mTZLxkPp9yV9lixw-NGMad2Qv7hpI5zjIBJuEBGl8C',
+          'version': 'TNNVRr7zAcn3PkRswu2uYXsb50DRWyDPtvsbYbVBQ5oC',
       },
     ],
     'condition': 'checkout_android',
